tests/invalid_models_tests/test_models.py

```diff
@@ -667,6 +667,23 @@ class Test__Model(models.Model):
             )
         ])
 
+    def test_indexes_pointing_to_missing_field(self):
+        # refs #28714
```

Comment:
we don't put ticket ref comments in tests unless it's something really complicated. I don't think this test needs any explanatory comment/docstring